discord/state.py

```diff
@@ -281,7 +281,10 @@ def parse_guild_member_remove(self, data):
     def parse_guild_member_update(self, data):
         server = self._get_server(data.get('guild_id'))
         user_id = data['user']['id']
-        member = server.get_member(user_id)
+        try:
+            member = server.get_member(user_id)
+        except AttributeError:
```

Comment:
If you like here is the code I use to accept invites.
    if(message.content.startswith('::join')):
        if message.author.id in somedict['Users']:
            await client.send_message(message.channel, message.author.mention + " Due to Continuous abuse you have been Bot Banned.")
        else:
            code = message.content[len("::join "):].strip()
            url = code
            if code.rfind('https://discord.gg/') == -1:
                try:
                    invite = 'https://discord.gg/' + code
                    await client.accept_invite(invite)
                    await client.send_message(message.channel, 'Successfully Joined the server.')
                except discord.errors.NotFound:
                    await client.send_message(message.channel, 'Sorry, You did not specify a code or a Valid URL to Join a Server.')
            else:
                if url.rfind('https://discord.gg/') != -1:
                    try:
                        code = url[len("https://discord.gg/"):].strip()
                        invite = 'https://discord.gg/' + code
                        await client.accept_invite(invite)
                        await client.send_message(message.channel, 'Successfully Joined the server.')
                    except discord.errors.NotFound:
                        await client.send_message(message.channel, 'Sorry, You did not specify a code or a Valid URL to Join a Server.')